app/reducers.js

```diff
@@ -97,8 +97,9 @@ const myActivities = (state = [], action) => {
 		case USER_ACTIVITY_LOADED:
 			return action.payload.activities;
 		case ADD_ACTIVITY:
+			return state;
```

Comment:
Why aren't we return the new array of activities?

Reply:
yeah, its probably best to just form the new array in the reducer. Right now the function is set up when you dispatch `add activity` it gives it the new array, which is probably not the best `redux` way to do it. I'll fix it up!